charts/jobs/templates/job.yaml

```diff
@@ -1,6 +1,7 @@
 {{- $v := .Values }} 
 {{- $ := . }}
 {{- $teamSecrets := (include "itemsByName" ($v.teamSecrets | default list) | fromYaml) }}
+{{- $podSecurityContext := get $v "podSecurityContext" | default (dict "runAsNonRoot" true "runAsUser" 1001 "runAsGroup" 1001) }}
```

Comment:
Sane defaults. Great. Is this found as default in the schema now too then?

Reply:
good point